src/pages/workspaces/workspace/Tools.js

```diff
@@ -183,13 +297,18 @@ export const Tools = _.flow(
         })
       ]),
       div({ style: styles.cardContainer(listView) }, [
-        _.map(config => {
-          return h(ToolCard, {
-            onCopy: () => this.setState({ copyingTool: { namespace: config.namespace, name: config.name } }),
-            onDelete: () => this.setState({ deletingTool: { namespace: config.namespace, name: config.name } }),
-            key: `${config.namespace}/${config.name}`, namespace, name, config, listView
-          })
-        }, configs),
+        h(Clickable, {
+          style: { ...styles.card, ...styles.shortCard, color: colors.green[0], fontSize: 18, lineHeight: '22px' },
+          onClick: () => this.setState({ findingTool: true })
+        }, [
+          'Find a Tool',
+          icon('plus-circle', { size: 32 })
+        ]),
+        listView ? div({ style: { flex: 1 } }, [tools]) : tools,
+        findingTool && h(FindToolModal, {
+          namespace, name,
+          onDismiss: () => this.setState({ findingTool: false })
+        }),
         configs && !configs.length && div(['No tools added']),
```

Comment:
When you click the "List" button, the `No tools added` text becomes right-aligned. Do we need this text anymore?
![Screen Shot 2019-03-11 at 11 08 04 AM](https://user-images.githubusercontent.com/2395211/54134218-f21fef00-43ed-11e9-9811-c78665349784.png)